plinth/modules/bind/forms.py

```diff
@@ -0,0 +1,45 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Forms for BIND module.
+"""
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from plinth.forms import ServiceForm
+
+from . import get_default
+
+
+class BindForm(ServiceForm):
+    """BIND configuration form"""
+    set_forwarding = forms.BooleanField(
+        label=_('Enable forwarding'),
+        required=False,
+        help_text=_('Enable forwarding on your BIND server'))
+
+    enable_dnssec = forms.BooleanField(
+        label=_('Enable DNSSEC'),
+        required=False,
+        help_text=_('Enable Domain Name System Security Extensions'))
+
+    dns_set = forms.CharField(
```

Comment:
I would propose to rename this to 'forwarders' or 'DNS forwarders'. I think that will explain it better.